### PR TITLE
Use s3 buckets for ExperimentAnalysisView

### PR DESCRIPTION
**Pull request by fbertsch**
This also refactors a bit, and changes the API - parallel changes are being made in telemetry-airflow.

### CODE REVIEW

File: src/main/scala/com/mozilla/telemetry/views/ExperimentAnalysisView.scala [hunk 1]
```diff
@@ -27,28 +33,44 @@ object ExperimentAnalysisView {
...
         .write.mode("overwrite").parquet(outputLocation)
     }
 
     spark.stop()
   }
 
+  def getInputLocation(conf: Conf): String = {
+    conf.outputBucket.get match {
+      case Some(b) => s"s3://${conf.inputBucket.get.getOrElse(b)}/${ExperimentSummaryView.jobName}/${ExperimentSummaryView.schemaVersion}"
+      case _ => conf.inputLocation.get.get
```
Comment (sunahsuh) [inline src/main/scala/com/mozilla/telemetry/views/ExperimentAnalysisView.scala]: I forgot to note -- `conf.inputLocation()` should have the same effect

File: src/main/scala/com/mozilla/telemetry/views/ExperimentAnalysisView.scala [hunk 2]
```diff
@@ -27,28 +33,44 @@ object ExperimentAnalysisView {
...
+    }
+  }
+
+  def getOutputLocation(conf: Conf): String = {
+    conf.outputBucket.get match {
+      case Some(b) => s"s3://$b/$JobName/$SchemaVersion"
+      case _ => conf.outputLocation.get.get
```
Comment (sunahsuh) [inline src/main/scala/com/mozilla/telemetry/views/ExperimentAnalysisView.scala]: Same here -- `conf.outputLocation()`

**Comment by fbertsch** [comment]
@sunahsuh agreed about having different variable naming schemes for different files, I'm trying to follow the [Scala Guidelines](http://docs.scala-lang.org/style/naming-conventions.html), but more important to have consistency. I'll fix it up later. RFAL!

**Comment by fbertsch** [comment]
@sunahsuh, do we want to close this PR, or should I rebase it?

**Comment by sunahsuh** [comment]
Oof, I think a rebase & fixing the conflicts at this point would be more work than just starting from scratch. Let's close this one.

**Pull request closed without merging**